cli/command/manifest/create_list.go

```diff
@@ -65,8 +65,13 @@ func createManifestList(dockerCli command.Cli, args []string, opts annotateOpts)
 	manifests := args[1:]
 	logrus.Debugf("retrieving digests of images...")
 	for _, manifestRef := range manifests {
+		namedRef, err := reference.ParseNormalizedNamed(manifestRef)
+		if err != nil {
+			// TODO: wrap error?
+			return err
+		}
 
-		mfstData, _, err := getImageData(dockerCli, manifestRef, targetRef.String(), false)
+		mfstData, _, err := getImageData(dockerCli, namedRef, targetRef.String(), false)
```

Comment:
hmm, this might be fixed by the second PR, where I factored out `normalizeReference()`.

I think we should be doing this normalization immediately when we receive data from the user. I noticed we had quite a bit of duplication because we kept repeating that operation. By keeping it to just a single place where we read input we should be able to avoid that.